R/SSmohnsrho.R

```diff
@@ -1,135 +1,183 @@
-#' Calculate Mohn's Rho values for select quantities
+#' Calculate Mohn's rho values for select quantities
 #'
 #' Function calculates:
-#' (1) a rho value for the ending year for each retrospective relative to the reference model
-#' as in Mohn (1999),
-#' (2) a "Wood's Hole Mohn's Rho", which is a rho value averaged across all years for each
-#' retrospective relative to the reference model, and
-#' (3) an "Alaska Fisheries Science Center and Hurtado-Ferro et al. (2015) Mohn's rho,
-#' which is the average rho per retrospective "peel".
+#' 1. a rho value for the ending year for each retrospective relative to the
+#'    reference model as in Mohn (1999);
+#' 1. a ``Wood's Hole Mohn's rho'', which is a rho value averaged across all
```

Comment:
It's new to me that you can create a numbered list without actually applying the correct numbering. Super useful.

Reply:
I am not sure if that functionality works for lists with (1) format, which is not standard in markdown, which is why I switched it to 1.